### PR TITLE
Features/240 flipud

### PR DESCRIPTION
**Pull request by mtar**
## Description

Include a summary of the change/s.
Please also include relevant motivation and context. List any dependencies that are required for this change.

Naïve implementation of flipud. It returns a new upside down tensor. Will have redundant code to #498.

Issue/s resolved: #240 

## Changes proposed:
- implement flipud

## Type of change

Remove irrelevant options:
- New feature (non-breaking change which adds functionality)
- Documentation update

## Due Diligence

- [x] All split configurations tested
- [x] Multiple dtypes tested in relevant functions
- [x] Documentation updated (if needed)
- [x] Updated changelog.md under the title "Pending Additions"

#### Does this change modify the behaviour of other functions? If so, which?
no

### CODE REVIEW

File: heat/core/manipulations.py [hunk 1]
```diff
@@ -556,6 +557,50 @@ def expand_dims(a, axis):
...
+    -------
+    res: ht.DNDarray
+        The flipped array.
+
+    Examples
+    --------
+    >>> a = ht.array([[0,1],[2,3]])
+    >>> ht.flipud(a)
+    tensor([[2, 3],
+        [0, 1]])
```
[the remark this answers is not in the document]
Comment (mtar) [reply heat/core/manipulations.py]: Example added.
I noticed that the multiprocess examples differ in the formatting. Is there a style guide?

[the remark this answers is not in the document]
Comment (coquelin77) [reply heat/core/manipulations.py]: there is no style guide on this issue. all that matters is that the processes are indicted

**Comment by coquelin77** [comment]
can you resolve the conflicts? the auto-merging doesnt play nice here

**Comment by mtar** [comment]
I was expecting that

**Comment by mtar** [comment]
I think the easiest way will be to simply call flip

**Comment by coquelin77** [comment]
> I think the easiest way will be to simply call flip

that makes sense to me. let me know when the changes are made

**Comment by mtar** [comment]
Done

**Pull request merged**